src/Icon/__tests__/Icon.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import 'jest-styled-components';
+
+import Icon from '..';
+import Theme from '../../Theme';
+
+describe('<Icon />', () => {
+  it('should render withouth a problem', () => {
+    const render = mount(<Icon theme={Theme} name="calendar" />);
+    expect(render).toMatchSnapshot();
+  });
+
+  it('should render with different name', () => {
+    const render = mount(<Icon theme={Theme} name="calendar" />);
```

Comment:
`should render with a different name` ? I think that `calendar` === `calendar` 😃